SourceEditorCompose/src/main/AndroidManifest.xml

```diff
@@ -18,13 +18,13 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.SourceEditorCompose"
+        android:theme="@style/Theme.ComposeSamples.NoActionBar"
```

Comment:
This one should be `Theme.ComposeSamples`, not `Theme.ComposeSamples.NoActionBar`.